resharper/resharper-unity/src/Yaml/UnityYamlEnabled.cs

```diff
@@ -0,0 +1,20 @@
+using JetBrains.Application.Settings;
+using JetBrains.DataFlow;
+using JetBrains.ProjectModel;
+using JetBrains.ProjectModel.DataContext;
+using JetBrains.ReSharper.Plugins.Unity.Settings;
+
+namespace JetBrains.ReSharper.Plugins.Unity.Yaml
+{
+    [SolutionComponent]
+    public class UnityYamlEnabled
+    {
+        public readonly IProperty<bool> YamlParsingEnabled;
+        public UnityYamlEnabled(Lifetime lifetime, ISolution solution, ISettingsStore settingsStore)
+        {
+            var settings = settingsStore.BindToContextLive(lifetime,
+                ContextRange.ManuallyRestrictWritesToOneContext(solution.ToDataContext()));
+            YamlParsingEnabled = settings.GetValueProperty(lifetime, (UnitySettings key) => key.EnableYamlParsing);
+        }
+    }
```

Comment:
This should probably do something to override/update `YamlSupport.EnableYamlParsing` - if that's been set to false in 2018.3.0, then it can't be set to true again in this update.

Reply:
I hope, that we can drop migration here. We will disable yaml parsing in most part of cases where performance can be affected via heuristic